gensim/models/wrappers/wordrank.py

```diff
@@ -113,6 +113,11 @@ def train(cls, wr_path, corpus_file, out_path, size=100, window=15, symmetric=1,
         with smart_open(meta_file, 'wb') as f:
             meta_info = "{0} {1}\n{2} {3}\n{4} {5}".format(numwords, numwords, numlines, cooccurrence_shuf_file, numwords, vocab_file)
             f.write(meta_info.encode('utf-8'))
+            
+        if iter % dump_period == 0:
+            iter += 1
+        else:
+            logger.warning('Resultant embedding would be from %d iteration', iter - iter % dump_period)
```

Comment:
Seeing this warning tells the user nothing -- can you rephrase/add context/suggestions what to do about it?

Also, `iteration` => `iterations` and `dump_period` as an argument to `warning()`, rather than formatted string.